### ISSUE TITLE
New shared Google Calendar not showing up

### ISSUE THREAD

**Issue by pburndorfer** [marcoancona/MineTime#161]
Recently I got a new shared calendar into my Google account. I already have multiple shared calendars in my account like my schedule for university etc. They all work fine also with MineTime.
Unfortunately the newly added one doesn't show up in MineTime. It seems as the calendar list in "Manage Accounts" doesn't refresh. Therefore I can't check the calendar to be visible because it's not in the list.
I can already view the calendar on calendar.google.com, so no fault on that side.

I'm running v 1.7.3. on Windows 10.

**Comment by marcoancona** [comment]
Yes, this is expected. The calendar list does not update automatically (yet). You can click on "Relink" for that.
